example/src/Examples/API/Images.tsx

```diff
@@ -52,6 +52,13 @@ export const Images = () => {
     [PAD, SIZE]
   );
 
+  useEffect(() => {
+    if (oslo) {
+      console.log("ImageInfo:", oslo?.getImageInfo());
+      console.log("Pixels:", oslo?.readPixels()?.slice(0, 10));
+    }
+  }, [oslo]);
+
```

Comment:
Would it be ok to remove this and use tests instead?
Here are examples of tests where we test API methods:
src/renderer/__tests__/e2e/RuntimeShader.spec.tsx
src/renderer/__tests__/e2e/Paths.spec.tsx
src/renderer/__tests__/e2e/AnimatedImages.spec.tsx